docs/src/biopsy.html.jinja2

```diff
@@ -296,6 +304,7 @@
             });
 
             //Biopsy
+            $("[name='bxp']").conditionallyEnable({sourceValues: "c", enable: $("#bxcr-container")}); //cancellation reason
```

Comment:
I would use conditionally display here instead. There's no real good reason to display this if any of the other options are selected. Changing to display would also negate the need to enable/disable validators. By default, they are only enabled if the field is visible.